packages/headless/src/features/facets/automatic-facet-set/interfaces/request.ts

```diff
@@ -2,5 +2,6 @@ import {FacetValue} from '../../../../product-listing.index';
 
 export type AutomaticFacetRequest = {
   field: string;
+  label: string;
   currentValues: FacetValue[];
 };
```

Comment:
Talked with jérémie to confirm that I was using the right request and response. I was missing the `label` in the request.